ipmiMgrApp/Db/afcv4.substitutions

```diff
@@ -1,20 +1,35 @@
-#nat>show_sensorinfo 6
+#nat> show_sensorinfo 6
 #Sensor Information for FRU 6 / AMC2
 #==================================================================
 #  #   SDRType  Sensor Entity Inst  Value   State    Name
 #------------------------------------------------------------------
-#  -   MDevLoc          0xc1  0x62                    AFC v4
+#  -   MDevLoc          0xc1  0x62                    AFC v4.0
 #  2   Compact  0xf2    0xc1  0x62  0x01              HOTSWAP AMC
-#  3   Full     Temp    0xc1  0x62  37 C       ok     TEMP FPGA
-#  4   Full     Temp    0xc1  0x62  24.5 C     ok     TEMP UC
-#  5   Full     Temp    0xc1  0x62  27.5 C     ok     TEMP CLK SWITCH
-#  6   Full     Temp    0xc1  0x62  35.5 C     ok     TEMP DCDC
-#  7   Full     Temp    0xc1  0x62  26.0 C     ok     TEMP RAM
-#  8   Compact  0xf0    0xc1  0x62  0x10              HS 006 AMC2
+#  3   Full     Voltage 0xc1  0x62  12.416 V   ok     AMC +12V
+#  4   Full     Voltage 0xc1  0x62  12.416 V   ok     RTM +12V
+#  5   Full     Voltage 0xc1  0x62  12.416 V   ok     FMC1 +12V
+#  6   Full     Voltage 0xc1  0x62  0.000 V    <=lnr  FMC1 VADJ
+#  7   Full     Voltage 0xc1  0x62  3.328 V    ok     FMC1 +3.3V
```

Comment:
Why AMC 12V and RTM 12V do not influence the `sensinst` values in the substitution?

Reply:
It does, in fact. I'll fix, thanks!